### ISSUE TITLE
Mark/UnMark any field as important

### ISSUE THREAD

**Issue by baziwane** [rapidftr/tracker#111]
This feature will allow an admin to mark a field as important. Once a field is marked as important a field worker would view a warning message if they leave it blank, but the form will still be saved.

**Comment by sferik** [comment]
@rdsubhas Is this something the @RapidRailsGirls team could work on? Does this involve changing the Android app or just the Rails app?

**Comment by rdsubhas** [comment]
@sferik I believe for this story:
- Administrator can edit a form field in the web app and mark it as Important
- Users can see warning messages for Important fields on the web app

And once this is done, I believe there is a separate story for the Android/API. Will confirm tomorrow... And yes it is definitely something that the @RapidRailsGirls team could pick up! Except we'll have to wait a bit, because the Forms are being revamped right now in #55, and will take a few days to complete that...
